src/cplscheme/ParallelCouplingScheme.hpp

```diff
@@ -8,6 +8,13 @@
 #include "m2n/SharedPointer.hpp"
 #include "utils/assertion.hpp"
 
+// Forward declaration to friend the boost test struct
+namespace CplSchemeTests {
+namespace ParallelImplicitCouplingSchemeTests {
+struct testInitializeData;
+}
+} // namespace CplSchemeTests
```

Comment:
I think we should go for a reusable solution as  in https://github.com/precice/precice/pull/921#issuecomment-870590088

Reply:
Done in 906dd95cac721658597d15b0862ad04ae4488f80.